# DEL ACERO AL ALGORITMO
## Estadística Descriptiva aplicada a la Educación Superior del Biobío

**Base de datos:** Matrículas de Educación Superior de la Región del Biobío, 2021.

### Pregunta central
**¿Desde qué base de capital humano parte el Biobío para transformar su histórica vocación industrial hacia una economía de manufactura avanzada e Industria 4.0?**

### ¿Qué significa el título “Del acero al algoritmo”?

El título funciona como una **metáfora del problema de investigación**.

- **“Acero”** representa la base productiva e industrial del Biobío y, dentro de este trabajo, se relaciona con los **motores productivos**: industria y manufactura, construcción e infraestructura, logística y puertos, forestal y madera, pesca y acuicultura, y agroalimentación.
- **“Algoritmo”** representa las **capacidades transformadoras** asociadas a una economía más tecnológica: Digital y TIC, automatización y robótica, y energía, sustentabilidad y biotecnología.

El análisis **no busca afirmar que el Biobío esté abandonando su industria ni demostrar que ya exista una transición causada por la tecnología**. La base entrega una fotografía de la matrícula de educación superior correspondiente a 2021. En este contexto, el título orienta el análisis hacia la comparación entre la formación vinculada con la base productiva y las capacidades tecnológicas que podrían complementarla.

En términos generales, la pregunta que guía el proyecto es: **¿con qué formación de capital humano contaba el Biobío en 2021 para combinar su identidad productiva con nuevas capacidades tecnológicas?**

El desarrollo se organiza desde la preparación de la base hasta la interpretación de los resultados, manteniendo como foco las variables y preguntas definidas para el análisis.


# 0. Carga y preparación de la base

La preparación de los datos comprende la carga del archivo, la revisión de su estructura y la construcción de las unidades de análisis necesarias para el trabajo: base original, matrículas de pregrado, registros con arancel positivo y ofertas académicas únicas.


In [1]:
#Importamos Pandas y cargamos el archivo Excel
import pandas as pd

df = pd.read_excel(
    '11_MATRICULAS_ED_SUPERIOR_BIOBIO_2021.xlsx',
    sheet_name='BASE DE DATOS'
)

#Mostramos las primeras filas para comprobar la carga
df.head()


,ID,GENERO,EDAD,RANGO EDAD,AÑO INGRESO,SEMESTRE INGRESO,TIPO DE INSTITUCION,NOMBRE DE INSTITUCION,ACREDITACION INSTITUCIONAL,PERIODO DE ACREDITACION,...,NIVEL CARRERA,AREA CONOCIMIENTO,DURACION PLAN DE ESTUDIO (SEMESTRES),DURACION PROCESO TITULACION (SEMESTRES),DURACION TOTAL CARRERA (SEMESTRES),VALOR MATRICULA (PESOS),VALOR ARANCEL (PESOS),REGION SEDE,PROVINCIA SEDE,COMUNA SEDE
0,52,Femenino,24,20 a 24,2016,Primer semestre,Universidades CRUCH,UNIVERSIDAD DE CONCEPCION,ACREDITADA,11/11/2016 AL 11/11/2023,...,Carreras Profesionales,Salud,10,0,10,120000,4243207,Biobio,Concepcion,Concepcion
1,58,Masculino,49,40 y mas,2019,Primer semestre,Institutos Profesionales,IP AIEP,ACREDITADA,22/10/2017 AL 22/10/2022,...,Carreras Tecnicas,Tecnologia,5,1,5,190000,1840000,Biobio,Concepcion,Concepcion
2,73,Masculino,21,20 a 24,2021,Primer semestre,Universidades Privadas,UNIVERSIDAD DEL DESARROLLO,ACREDITADA,15/12/2016 AL 15/12/2021,...,Carreras Profesionales,Derecho,10,2,12,494445,5351640,Biobio,Concepcion,Concepcion
3,80,Femenino,24,20 a 24,2020,Primer semestre,Centros de Formacion Tecnica,CFT LOTA-ARAUCO,ACREDITADA,21/07/2019 AL 21/07/2023,...,Carreras Tecnicas,Tecnologia,5,1,5,30000,1450000,Biobio,Concepcion,Lota
4,88,Femenino,25,25 a 29,2018,Primer semestre,Universidades CRUCH,UNIVERSIDAD DEL BIO-BIO,ACREDITADA,28/08/2019 AL 28/08/2024,...,Carreras Profesionales,Administracion y Comercio,10,0,10,105000,3255000,Biobio,Concepcion,Concepcion


**Análisis e interpretación:** Se cargó la hoja **BASE DE DATOS** del archivo de matrículas mediante pandas y se realizó una revisión inicial con `head()`. Esta comprobación permitió verificar que los registros se importaron correctamente y que la base contiene las variables necesarias para el estudio, entre ellas género, edad, tipo de institución, carrera, área del conocimiento, arancel y provincia de sede. Esta revisión inicial es relevante porque confirma que la información disponible permite desarrollar tanto la descripción general de la matrícula como las tres preguntas específicas del trabajo.


In [2]:
#Revisamos la cantidad de filas y columnas
df.shape


(106555, 28)

**Análisis e interpretación:** La base original contiene **106.555 registros y 28 columnas**. Esta cifra establece el punto de partida del análisis y permite dimensionar la información disponible antes de aplicar filtros. Posteriormente, el número de registros disminuye de acuerdo con el objetivo de cada análisis: primero se selecciona el pregrado y, para los cálculos de precios, se distinguen los registros con arancel positivo y las ofertas académicas únicas.


In [3]:
#Revisamos las columnas, tipos de datos y valores no nulos
df.info()


<class 'pandas.DataFrame'>
RangeIndex: 106555 entries, 0 to 106554
Data columns (total 28 columns):
 #   Column                                   Non-Null Count   Dtype  
---  ------                                   --------------   -----  
 0   ID                                       106555 non-null  int64  
 1   GENERO                                   106555 non-null  str    
 2   EDAD                                     106555 non-null  int64  
 3   RANGO EDAD                               106555 non-null  str    
 4   AÑO INGRESO                              106555 non-null  int64  
 5   SEMESTRE INGRESO                         106555 non-null  str    
 6   TIPO DE INSTITUCION                      106555 non-null  str    
 7   NOMBRE DE INSTITUCION                    106555 non-null  str    
 8   ACREDITACION INSTITUCIONAL               106555 non-null  str    
 9   PERIODO DE ACREDITACION                  102840 non-null  str    
 10  AÑOS DE ACREDITACION                     10

**Análisis e interpretación:** La revisión de la estructura de la base confirma las **106.555 filas y 28 variables**, además de mostrar el tipo de dato y la cantidad de valores disponibles en cada columna. Se observan datos faltantes en **PERIODO DE ACREDITACION** y **AÑOS DE ACREDITACION**, mientras que las variables centrales de este estudio, como edad, carrera, área del conocimiento, arancel, institución y provincia, presentan información completa. Por lo tanto, los valores faltantes detectados no afectan directamente los análisis principales desarrollados en el trabajo.


In [4]:
#Controlamos duplicados completos y valores nulos
duplicados_completos = int(df.duplicated().sum())
nulos_por_columna = df.isna().sum()
nulos_relevantes = nulos_por_columna[nulos_por_columna > 0]

print('Duplicados completos:', duplicados_completos)
print('\nValores nulos por columna:')
print(nulos_relevantes)


Duplicados completos: 0

Valores nulos por columna:
PERIODO DE ACREDITACION    3715
AÑOS DE ACREDITACION       3715
dtype: int64


**Control de calidad e interpretación:** La revisión no detectó filas completamente duplicadas. Los valores faltantes se concentran en **PERIODO DE ACREDITACION** y **AÑOS DE ACREDITACION**, con **3.715 registros sin información en cada una**, equivalentes a cerca de 3,5% de la base original. Estas dos variables no se utilizan para responder las preguntas centrales del trabajo, por lo que sus valores faltantes no afectan directamente los cálculos posteriores. Para los análisis de precios también se excluyen **26 registros con arancel igual a \$0**; esta exclusión se aplica únicamente para trabajar con aranceles positivos comparables y no permite inferir por qué esos registros presentan valor cero.


In [5]:
#Contamos los niveles de carrera presentes en la base
df['NIVEL CARRERA'].value_counts()


NIVEL CARRERA
Carreras Profesionales    75855
Carreras Tecnicas         25238
Magister                   3340
Postitulo                  1710
Doctorado                   412
Name: count, dtype: int64

**Análisis e interpretación:** La distribución por nivel de carrera muestra que la mayor parte de los registros corresponde a **Carreras Profesionales (75.855)** y **Carreras Técnicas (25.238)**. También existen registros de Magíster, Postítulo y Doctorado, pero estos representan niveles formativos distintos del foco principal del estudio. Por esta razón, el análisis posterior se concentra en carreras profesionales y técnicas, de modo que la comparación se realice sobre una población de pregrado más homogénea.


La base original contiene **106.555 registros y 28 columnas**. Para el análisis principal nos interesa trabajar con carreras de pregrado.


In [6]:
#Definimos los niveles de pregrado
pregrado = ['Carreras Profesionales', 'Carreras Tecnicas']

#Filtramos las matrículas de pregrado
d = df[
    df['NIVEL CARRERA'].isin(pregrado)
].copy()

#Para analizar precios dejamos solo aranceles mayores que $0
d_precio = d[
    d['VALOR ARANCEL (PESOS)'] > 0
].copy()

print('Matrículas de pregrado:', d.shape[0])
print('Matrículas con arancel mayor que $0:', d_precio.shape[0])


Matrículas de pregrado: 101093
Matrículas con arancel mayor que $0: 101067


**Análisis e interpretación:** Al considerar únicamente **Carreras Profesionales** y **Carreras Técnicas**, la base queda compuesta por **101.093 matrículas de pregrado**. Para los análisis de precios se aplica además la condición de arancel mayor que cero, con lo que se obtienen **101.067 registros**. La diferencia corresponde a 26 matrículas con arancel igual a cero. Estos casos se mantienen fuera de los cálculos de precios porque el objetivo es comparar aranceles positivos registrados; la base no entrega información suficiente para determinar la razón de esos valores cero.


## Oferta académica única

Cuando analizamos precios no conviene repetir el mismo arancel una vez por cada estudiante matriculado. Por eso creamos una base donde cada oferta académica aparece una sola vez.


In [7]:
#Columnas que identifican una oferta académica
clave_oferta = [
    'NOMBRE DE INSTITUCION',
    'NOMBRE CARRERA',
    'COMUNA SEDE',
    'MODALIDAD',
    'JORNADA',
    'DURACION TOTAL CARRERA (SEMESTRES)',
    'VALOR MATRICULA (PESOS)',
    'VALOR ARANCEL (PESOS)'
]

#Eliminamos las ofertas repetidas
ofertas = d_precio.drop_duplicates(
    subset=clave_oferta
).copy()

print('Ofertas académicas únicas:', ofertas.shape[0])


Ofertas académicas únicas: 1273


**Análisis e interpretación:** Para comparar aranceles se construyó una base de **1.273 ofertas académicas únicas**. La base original está organizada a nivel de matrícula, por lo que una misma carrera puede repetir el mismo arancel tantas veces como estudiantes tenga inscritos. Mantener esas repeticiones daría más peso a las carreras con mayor matrícula, aunque su precio fuera exactamente el mismo. Por ello, se consideró como una misma oferta la combinación de institución, carrera, comuna, modalidad, jornada, duración, valor de matrícula y arancel. Esta decisión permite que las comparaciones de precios representen ofertas académicas y no cantidad de estudiantes.


# 1. Población, base de análisis y variables

**Población de interés:** matrículas de educación superior de la Región del Biobío durante 2021.

**Base disponible:** 106.555 registros.

**Subbase principal:** 101.093 matrículas de pregrado.

**Unidad de análisis:** para describir la matrícula se trabaja con registros de matrícula; para comparar precios entre carreras y áreas se utilizan ofertas académicas únicas. No se extrajo una muestra aleatoria adicional, sino que se trabajó con la base entregada y con subbases construidas según el objetivo de cada análisis.

## 1.1 Clasificación de las variables principales

Para organizar el análisis se clasificaron las variables principales según su naturaleza estadística. Las variables cualitativas representan categorías, mientras que las cuantitativas expresan magnitudes numéricas. A su vez, se distinguen los subtipos nominal, ordinal, discreta y continua.

| Variable | Tipo general | Subtipo | Justificación breve |
|---|---|---|---|
| GÉNERO | Cualitativa | Nominal | Presenta categorías sin un orden natural. |
| RANGO EDAD | Cualitativa | Ordinal | Sus categorías siguen un orden de menor a mayor edad. |
| TIPO DE INSTITUCIÓN | Cualitativa | Nominal | Distingue categorías de institución sin jerarquía numérica. |
| NOMBRE CARRERA | Cualitativa | Nominal | Identifica carreras como categorías sin un orden natural. |
| ÁREA DEL CONOCIMIENTO | Cualitativa | Nominal | Agrupa campos de estudio sin establecer jerarquía entre ellos. |
| PROVINCIA SEDE | Cualitativa | Nominal | Identifica territorios sin un orden estadístico. |
| EDAD | Cuantitativa | Continua | Conceptualmente mide edad sobre una escala continua, aunque en la base se registra en años enteros. |
| AÑO INGRESO | Cuantitativa | Discreta | Se registra mediante años enteros, por ejemplo 2019, 2020 o 2021. |
| DURACIÓN TOTAL CARRERA (SEMESTRES) | Cuantitativa | Continua | Se trabaja como una medida de duración y la base la expresa en semestres. |
| VALOR ARANCEL (PESOS) | Cuantitativa | Continua | Representa una magnitud monetaria sobre la que se calculan medidas descriptivas. |

**Criterio utilizado:** nominal corresponde a categorías sin orden; ordinal a categorías ordenadas; discreta a valores numéricos separados; y continua a mediciones realizadas sobre una escala numérica.


# 2. Descripción general de la base

Antes de responder las preguntas del trabajo revisamos algunas características generales usando tablas de frecuencia, gráficos y medidas descriptivas.


## 2.1 Género


In [8]:
#Calculamos la frecuencia absoluta
cuenta_genero = d.groupby('GENERO').size()

#Creamos la tabla con frecuencia absoluta y relativa
tabla_genero = pd.DataFrame({
    'Frecuencia absoluta': cuenta_genero,
    'Frecuencia relativa (%)': round(cuenta_genero/d.shape[0] * 100, 1)
})

tabla_genero


,Frecuencia absoluta,Frecuencia relativa (%)
GENERO,,
Femenino,54881,54.3
Masculino,46212,45.7


**Análisis e interpretación:** La distribución por género muestra **54.881 matrículas femeninas (54,3%)** y **46.212 masculinas (45,7%)** dentro de las 101.093 matrículas de pregrado. La diferencia es de 8,6 puntos porcentuales, por lo que la participación femenina es mayor en el conjunto analizado. Este resultado describe la composición general de la matrícula, pero por sí solo no permite explicar las razones de esa diferencia ni asumir que el mismo patrón se repite en todas las áreas del conocimiento.


En la base de pregrado, las matrículas femeninas representan **54,3%** y las masculinas **45,7%**.


## 2.2 Área del conocimiento


In [9]:
#Calculamos la frecuencia por área del conocimiento
cuenta_area = d.groupby('AREA CONOCIMIENTO').size()

#Creamos la tabla y la ordenamos de mayor a menor
tabla_area = pd.DataFrame({
    'Frecuencia absoluta': cuenta_area,
    'Frecuencia relativa (%)': round(cuenta_area/d.shape[0] * 100, 1)
})

tabla_area = tabla_area.sort_values(
    'Frecuencia absoluta',
    ascending=False
)

tabla_area


,Frecuencia absoluta,Frecuencia relativa (%)
AREA CONOCIMIENTO,,
Tecnologia,27866,27.6
Salud,24597,24.3
Administracion y Comercio,13526,13.4
Educacion,11480,11.4
Ciencias Sociales,10677,10.6
Arte y Arquitectura,3611,3.6
Derecho,3529,3.5
Ciencias Basicas,2936,2.9
Agropecuaria,2361,2.3


**Análisis e interpretación:** La matrícula se concentra principalmente en **Tecnología**, con **27.866 registros (27,6%)**, y **Salud**, con **24.597 (24,3%)**. En conjunto, ambas áreas reúnen **51,9%** de las matrículas de pregrado. Si se agregan Administración y Comercio (13,4%) y Educación (11,4%), las cuatro áreas principales representan aproximadamente **76,7%** del total. Esto muestra que la distribución no es uniforme entre áreas y que una parte importante de la formación superior regional se concentra en un número reducido de campos de estudio.


Las áreas con mayor cantidad de matrículas son **Tecnología** y **Salud**.


## 2.3 Edad agrupada


In [10]:
#Agrupamos la edad en intervalos
d['Intervalos_edad'] = pd.cut(
    d['EDAD'],
    bins=10,
    include_lowest=True,
    precision=0
)

#Calculamos las frecuencias
cuenta_edad = d.groupby(
    'Intervalos_edad',
    observed=True
).size()

tabla_edad = pd.DataFrame({
    'Frecuencia absoluta': cuenta_edad,
    'Frecuencia relativa (%)': cuenta_edad/d.shape[0] * 100
})

#Como los intervalos están ordenados, agregamos las frecuencias acumuladas
tabla_edad['Frecuencia acumulada'] = tabla_edad['Frecuencia absoluta'].cumsum()
tabla_edad['Frecuencia relativa acumulada (%)'] = tabla_edad['Frecuencia relativa (%)'].cumsum()

round(tabla_edad, 1)


,Frecuencia absoluta,Frecuencia relativa (%),Frecuencia acumulada,Frecuencia relativa acumulada (%)
Intervalos_edad,,,,
"(16.0, 23.0]",48400,47.9,48400,47.9
"(23.0, 29.0]",35920,35.5,84320,83.4
"(29.0, 35.0]",9489,9.4,93809,92.8
"(35.0, 41.0]",3706,3.7,97515,96.5
"(41.0, 46.0]",2061,2.0,99576,98.5
"(46.0, 52.0]",1072,1.1,100648,99.6
"(52.0, 58.0]",362,0.4,101010,99.9
"(58.0, 64.0]",73,0.1,101083,100.0
"(64.0, 70.0]",7,0.0,101090,100.0


**Análisis e interpretación:** La edad se agrupó en diez intervalos para resumir una variable con numerosos valores individuales y facilitar la comparación de su distribución. El primer intervalo, que reúne aproximadamente las edades de **17 a 23 años**, concentra **47,9%** de las matrículas. El segundo intervalo, correspondiente a edades **mayores de 23 y hasta 29 años**, representa **35,5%**. En conjunto, ambos intervalos acumulan **83,4%** del total, por lo que la mayor parte de las matrículas de pregrado se concentra entre los 17 y 29 años. La frecuencia acumulada permite observar cómo aumenta el porcentaje total al avanzar desde los grupos de menor edad hacia los de mayor edad.


## 2.4 Arancel agrupado


In [11]:
#Agrupamos los aranceles en intervalos
d_precio['Intervalos_arancel'] = pd.cut(
    d_precio['VALOR ARANCEL (PESOS)'],
    bins=9,
    include_lowest=True,
    precision=0
)

#Calculamos las frecuencias
cuenta_arancel = d_precio.groupby(
    'Intervalos_arancel',
    observed=True
).size()

tabla_arancel = pd.DataFrame({
    'Frecuencia absoluta': cuenta_arancel,
    'Frecuencia relativa (%)': cuenta_arancel/d_precio.shape[0] * 100
})

#Como los intervalos están ordenados, agregamos las frecuencias acumuladas
tabla_arancel['Frecuencia acumulada'] = tabla_arancel['Frecuencia absoluta'].cumsum()
tabla_arancel['Frecuencia relativa acumulada (%)'] = tabla_arancel['Frecuencia relativa (%)'].cumsum()

round(tabla_arancel, 1)


,Frecuencia absoluta,Frecuencia relativa (%),Frecuencia acumulada,Frecuencia relativa acumulada (%)
Intervalos_arancel,,,,
"(641865.0, 1553741.0]",10846,10.7,10846,10.7
"(1553741.0, 2457482.0]",32641,32.3,43487,43.0
"(2457482.0, 3361223.0]",11719,11.6,55206,54.6
"(3361223.0, 4264964.0]",20615,20.4,75821,75.0
"(4264964.0, 5168706.0]",17271,17.1,93092,92.1
"(5168706.0, 6072447.0]",4193,4.1,97285,96.3
"(6072447.0, 6976188.0]",1462,1.4,98747,97.7
"(6976188.0, 7879929.0]",1257,1.2,100004,98.9
"(7879929.0, 8783670.0]",1063,1.1,101067,100.0


**Análisis e interpretación:** Los aranceles se agruparon en intervalos para observar cómo se distribuyen los **101.067 registros de pregrado con arancel mayor que cero**. El tramo con mayor frecuencia se encuentra aproximadamente entre **\$1,55 y \$2,46 millones**, con **32,3%** de los registros. También se observa una presencia relevante en los intervalos de **\$3,36 a \$4,26 millones (20,4%)** y de **\$4,26 a \$5,17 millones (17,1%)**. La distribución muestra que los aranceles no se concentran en un único nivel de precio y que existen diferencias importantes entre los distintos tramos, lo que justifica profundizar posteriormente en las ofertas de mayor valor.


## 2.5 Gráficos principales


In [12]:
#Importamos la biblioteca para crear gráficos
import matplotlib.pyplot as plt


**Análisis e interpretación:** Para complementar las tablas con una lectura visual se incorporó `matplotlib.pyplot`. Los gráficos se utilizan como apoyo para comparar categorías y distribuciones que, en una tabla extensa, pueden ser menos evidentes a simple vista. En las siguientes secciones se emplean principalmente gráficos de barras, ya que permiten contrastar de forma directa magnitudes entre áreas, intervalos de edad y grupos definidos en el análisis.


In [13]:
#Gráfico de barras por área del conocimiento
fig, ax = plt.subplots(figsize=(10,5))

ax.bar(
    tabla_area.index,
    tabla_area['Frecuencia absoluta'],
    edgecolor='black'
)

ax.set_title('Matrículas por área del conocimiento')
ax.set_xlabel('Área del conocimiento')
ax.set_ylabel('Cantidad de matrículas')
plt.xticks(rotation=45, ha='right')

plt.show()


**Análisis e interpretación:** El gráfico de matrículas por área del conocimiento confirma visualmente la concentración observada en la tabla de frecuencias. **Tecnología** presenta la barra más alta, seguida por **Salud**, mientras que áreas como Humanidades, Agropecuaria y Ciencias Básicas registran cantidades considerablemente menores. La visualización permite identificar con rapidez las diferencias de tamaño entre áreas y refuerza la conclusión de que la matrícula regional se concentra especialmente en Tecnología y Salud.


In [14]:
#Gráfico de la distribución de la edad
fig, ax = plt.subplots(figsize=(8,5))

ax.bar(
    tabla_edad.index.astype(str),
    tabla_edad['Frecuencia relativa (%)'],
    edgecolor='black',
    width=1
)

ax.set_title('Distribución de matrículas por edad')
ax.set_xlabel('Intervalos de edad')
ax.set_ylabel('Frecuencia relativa (%)')
plt.xticks(rotation=45, ha='right')

plt.show()


**Análisis e interpretación:** La distribución gráfica de la edad muestra una concentración marcada en los primeros intervalos. Los grupos más jóvenes reúnen la mayor proporción de matrículas y, a medida que aumenta la edad, la participación disminuye de forma importante. Este patrón coincide con la tabla de frecuencias: aproximadamente **83,4%** de las matrículas se acumula hasta los **29 años**. Por lo tanto, la composición etaria del pregrado analizado es predominantemente joven, aunque la base también registra estudiantes en rangos de edad superiores.


In [15]:
#Gráfico de la distribución del arancel
fig, ax = plt.subplots(figsize=(9,5))

ax.bar(
    tabla_arancel.index.astype(str),
    tabla_arancel['Frecuencia relativa (%)'],
    edgecolor='black',
    width=1
)

ax.set_title('Distribución de matrículas según arancel')
ax.set_xlabel('Intervalos de arancel')
ax.set_ylabel('Frecuencia relativa (%)')
plt.xticks(rotation=45, ha='right')

plt.show()


**Análisis e interpretación:** El gráfico de aranceles muestra que la mayor concentración de registros se encuentra en rangos intermedios, especialmente alrededor de **\$1,5 a \$2,5 millones**, pero también existen grupos relevantes en tramos superiores. Las frecuencias disminuyen hacia los valores más altos, aunque estos no desaparecen. Esta forma de la distribución es importante para el análisis posterior, ya que confirma la existencia de una zona superior de aranceles que puede estudiarse por separado mediante un criterio estadístico reproducible.


Los gráficos muestran que la matrícula se concentra principalmente en **Tecnología y Salud**, en estudiantes de edades más jóvenes y en los rangos medios de arancel.


# 3. Medidas descriptivas

Las variables cuantitativas se resumen mediante medidas de tendencia central y dispersión. Estas medidas permiten describir tanto el valor central de las distribuciones como el grado de variación existente entre los registros.


## 3.1 Arancel


In [16]:
#Calculamos media, mediana y moda del arancel
media_arancel = d_precio['VALOR ARANCEL (PESOS)'].mean()
mediana_arancel = d_precio['VALOR ARANCEL (PESOS)'].median()
moda_arancel = d_precio['VALOR ARANCEL (PESOS)'].mode()[0]

print(f'Media: ${media_arancel:.0f}')
print(f'Mediana: ${mediana_arancel:.0f}')
print(f'Moda: ${moda_arancel:.0f}')


Media: $3224895
Mediana: $3133750
Moda: $3373000


**Análisis e interpretación:** A nivel de las **101.067 matrículas de pregrado con arancel mayor que cero**, la media del arancel es aproximadamente **\$3.224.895**, la mediana **\$3.133.750** y la moda **\$3.373.000**. La media representa el promedio de los aranceles asociados a las matrículas, la mediana divide esos registros en dos mitades y la moda corresponde al valor que más se repite. La media es ligeramente superior a la mediana, lo que es compatible con la presencia de valores altos que elevan el promedio. Estas medidas describen la distribución del arancel a nivel de matrícula; cuando se comparan precios entre áreas en la Pregunta 1 se utilizan, en cambio, las **1.273 ofertas académicas únicas** para evitar que una carrera con más estudiantes tenga mayor peso solo por su matrícula.


## 3.2 Edad


In [17]:
#Calculamos media, mediana y moda de la edad
media_edad = d['EDAD'].mean()
mediana_edad = d['EDAD'].median()
moda_edad = d['EDAD'].mode()[0]

print(f'Media: {media_edad:.1f} años')
print(f'Mediana: {mediana_edad:.0f} años')
print(f'Moda: {moda_edad} años')


Media: 24.4 años
Mediana: 23 años
Moda: 19 años


**Análisis e interpretación:** La edad presenta una **media de 24,4 años**, una **mediana de 23 años** y una **moda de 19 años**. La mediana indica que la mitad de las matrículas corresponde a personas de 23 años o menos, mientras que 19 años es la edad individual que aparece con mayor frecuencia. Estos resultados son coherentes con la distribución por intervalos, donde los primeros grupos concentran la mayor parte de los registros. En conjunto, las medidas de tendencia central confirman que la matrícula de pregrado analizada está compuesta principalmente por estudiantes jóvenes.


## 3.3 Dispersión del arancel


In [18]:
#Calculamos máximo y mínimo
maximo_arancel = d_precio['VALOR ARANCEL (PESOS)'].max()
minimo_arancel = d_precio['VALOR ARANCEL (PESOS)'].min()

#Calculamos el rango
rango_arancel = maximo_arancel - minimo_arancel

#Calculamos la desviación estándar
desviacion_arancel = d_precio['VALOR ARANCEL (PESOS)'].std()

#Calculamos el coeficiente de variación
cv_arancel = desviacion_arancel / media_arancel * 100

print(f'Rango: ${rango_arancel:.0f}')
print(f'Desviación estándar: ${desviacion_arancel:.0f}')
print(f'CV: {cv_arancel:.1f}%')

Rango: $8133670
Desviación estándar: $1518991
CV: 47.1%


**Análisis e interpretación:** La dispersión del arancel se calculó sobre las **101.067 matrículas con arancel mayor que cero**. El rango es de **\$8.133.670**, por lo que existe una distancia amplia entre el menor y el mayor valor registrado. La desviación estándar alcanza aproximadamente **\$1.518.991**, mientras que el **coeficiente de variación es 47,1%**. Este último resultado significa que la desviación estándar equivale aproximadamente al 47,1% de la media del arancel. En conjunto, las medidas muestran que los valores no se encuentran concentrados estrechamente alrededor de un único precio central, por lo que es útil complementar el promedio con la mediana y con medidas de posición.


El **coeficiente de variación de 47,1%** indica que la desviación estándar representa casi la mitad del arancel promedio. Esto evidencia una dispersión relevante entre los aranceles asociados a las matrículas y explica por qué, en los análisis de precios, no se utiliza únicamente la media como medida de referencia.


# 4. Aplicación regional: Del acero al algoritmo

Para responder la pregunta central clasificamos las carreras en nueve familias. Esta clasificación fue creada para el trabajo y no corresponde a una clasificación oficial.

**Motores productivos:** industria, construcción, logística, forestal, pesca y agroalimentario.

**Capacidades transformadoras:** Digital/TIC, automatización/robótica y energía/sustentabilidad/biotecnología.


In [19]:
#Primero dejamos todas las carreras como "Otros campos"
d['ADN_BIOBIO'] = 'Otros campos'


**Análisis e interpretación:** Para construir la clasificación regional se creó la variable **ADN_BIOBIO** y se asignó inicialmente la categoría **“Otros campos”** a todas las matrículas. A partir de esa base se fueron identificando las carreras que cumplían los criterios definidos para cada familia productiva o transformadora. Este procedimiento permite conservar una categoría residual para aquellas carreras que no forman parte del marco “Del acero al algoritmo”, evitando forzar su incorporación en grupos con los que no existe una relación directa dentro de la clasificación propuesta.


In [20]:
#Clasificamos las familias asociadas a motores productivos

#1. Industria y manufactura
industria = [
    "DISENO INDUSTRIAL",
    "ELECTRICIDAD INDUSTRIAL MENCION INSTALACIONES ELECTRICAS",
    "ELECTROMECANICA",
    "FABRICACION Y MONTAJE INDUSTRIAL",
    "INGENIERIA CIVIL DE MATERIALES",
    "INGENIERIA CIVIL EN METALURGIA",
    "INGENIERIA CIVIL INDUSTRIAL",
    "INGENIERIA CIVIL INDUSTRIAL - CONTINUIDAD INGENIEROS NO AFIN",
    "INGENIERIA CIVIL INDUSTRIAL, PROGRAMA CONTINUIDAD DE ESTUDIOS",
    "INGENIERIA CIVIL MECANICA",
    "INGENIERIA CIVIL METALURGICA",
    "INGENIERIA CIVIL QUIMICA",
    "INGENIERIA DE EJECUCION EN ELECTRICIDAD",
    "INGENIERIA DE EJECUCION EN ELECTRICIDAD-PE",
    "INGENIERIA DE EJECUCION EN GESTION INDUSTRIAL",
    "INGENIERIA DE EJECUCION EN MANTENIMIENTO INDUSTRIAL",
    "INGENIERIA DE EJECUCION EN MANTENIMIENTO INDUSTRIAL-PE",
    "INGENIERIA DE EJECUCION EN MECANICA",
    "INGENIERIA DE EJECUCION EN MECANICA AUTOMOTRIZ Y MAQUINARIA PESADA-PE",
    "INGENIERIA DE EJECUCION INDUSTRIAL",
    "INGENIERIA DE EJECUCION INDUSTRIAL PLAN CONTINUIDAD",
    "INGENIERIA DE EJECUCION INDUSTRIAL-PE",
    "INGENIERIA DE EJECUCION MECANICA",
    "INGENIERIA DE EJECUCION MECANICA DE PROCESOS Y MANTENIMIENTO INDUSTRIAL",
    "INGENIERIA DE EJECUCION MECANICA-PE",
    "INGENIERIA EN ELECTRICIDAD MENCION POTENCIA",
    "INGENIERIA EN ELECTRICIDAD MENCION PROYECTOS DE INSTALACIONES ELECTRICAS",
    "INGENIERIA EN MANTENIMIENTO INDUSTRIAL",
    "INGENIERIA EN MAQUINARIA PESADA Y VEHICULOS AUTOMOTRICES",
    "INGENIERIA EN MAQUINARIA, VEHICULOS AUTOMOTRICES Y SISTEMAS ELECTRONICOS",
    "INGENIERIA EN MECANICA AUTOMOTRIZ Y AUTOTRONICA",
    "INGENIERIA EN MECANICA Y ELECTROMOVILIDAD AUTOMOTRIZ",
    "INGENIERIA INDUSTRIAL",
    "INGENIERIA MECANICA EN MANTENIMIENTO INDUSTRIAL",
    "MANTENIMIENTO INDUSTRIAL",
    "MAQUINARIA PESADA",
    "MECANICA AUTOMOTRIZ EN MAQUINARIA PESADA",
    "MECANICA AUTOMOTRIZ EN SISTEMAS ELECTRONICOS",
    "MECANICA DE EQUIPO PESADO",
    "MECANICA INDUSTRIAL",
    "PROGRAMA DE CONTINUIDAD DE ESTUDIOS EN INGENIERIA DE EJECUCION INDUSTRIAL",
    "REFRIGERACION INDUSTRIAL Y AIRE ACONDICIONADO",
    "REFRIGERACION INDUSTRIAL Y CLIMATIZACION",
    "TECNICO DE NIVEL SUPERIOR EN MANTENCION INDUSTRIAL",
    "TECNICO DE NIVEL SUPERIOR EN MAQUINARIA NAVAL",
    "TECNICO EN ELECTRICIDAD",
    "TECNICO EN ELECTRICIDAD INDUSTRIAL",
    "TECNICO EN ELECTROMECANICA",
    "TECNICO EN FABRICACION Y MONTAJE INDUSTRIAL",
    "TECNICO EN MANTENIMIENTO INDUSTRIAL",
    "TECNICO EN MECANICA AUTOMOTRIZ Y AUTOTRONICA",
    "TECNICO EN MECANICA AUTOMOTRIZ Y MAQUINARIA PESADA",
    "TECNICO EN MECANICA AUTOMOTRIZ Y SISTEMAS ELECTRICOS",
    "TECNICO EN MECANICA Y ELECTROMOVILIDAD AUTOMOTRIZ",
    "TECNICO QUIMICO ANALISTA INDUSTRIAL",
    "TECNICO UNIVERSITARIO EN ELECTRICIDAD",
    "TECNICO UNIVERSITARIO EN MANTENIMIENTO INDUSTRIAL",
    "TECNICO UNIVERSITARIO EN MECANICA AUTOMOTRIZ",
    "TECNICO UNIVERSITARIO EN MECANICA INDUSTRIAL",
    "TECNICO UNIVERSITARIO EN QUIMICA CON MENCION EN QUIMICA INDUSTRIAL"
]

d.loc[
    d['NOMBRE CARRERA'].isin(industria),
    'ADN_BIOBIO'
] = 'Industria y manufactura'


#2. Construcción e infraestructura
construccion = [
    "ARQUITECTURA",
    "CONSTRUCCION",
    "CONSTRUCCION CIVIL",
    "CONSTRUCCION CIVIL (PLAN CONTINUIDAD)",
    "DIBUJO Y MODELAMIENTO ARQUITECTONICO Y ESTRUCTURAL",
    "INGENIERIA CIVIL",
    "INGENIERIA CIVIL-PLAN COMUN",
    "INGENIERIA EN CONSTRUCCION",
    "INGENIERIA EN CONSTRUCCION-PE",
    "PROGRAMA DE CONTINUIDAD DE ESTUDIOS EN CONSTRUCCION CIVIL",
    "TECNICO DE NIVEL SUPERIOR EN CONSTRUCCION",
    "TECNICO DE NIVEL SUPERIOR EN CONSTRUCCION CIVIL",
    "TECNICO EN CONSTRUCCION",
    "TECNICO EN CONSTRUCCIONES CIVILES",
    "TECNICO UNIVERSITARIO EN CONSTRUCCION"
]

d.loc[
    d['NOMBRE CARRERA'].isin(construccion),
    'ADN_BIOBIO'
] = 'Construccion e infraestructura'


#3. Logística y puertos
logistica = [
    "INGENIERIA CIVIL LOGISTICA",
    "INGENIERIA DE EJECUCION EN GESTION LOGISTICA-PE",
    "INGENIERIA DE EJECUCION EN LOGISTICA MARITIMA PORTUARIA",
    "INGENIERIA DE EJECUCION EN LOGISTICA MARITIMA PORTUARIA-PE",
    "INGENIERIA DE EJECUCION INDUSTRIAL MENCION MANTENIMIENTO Y LOGISTICA",
    "INGENIERIA EN COMERCIO EXTERIOR",
    "INGENIERIA EN GESTION LOGISTICA",
    "INGENIERIA EN LOGISTICA OPERATIVA",
    "INGENIERIA EN LOGISTICA Y TRANSPORTE",
    "LOGISTICA",
    "LOGISTICA Y OPERACIONES INDUSTRIALES",
    "TECNICO DE NIVEL SUPERIOR EN GESTION LOGISTICA",
    "TECNICO DE NIVEL SUPERIOR EN LOGISTICA Y OPERACIONES INDUSTRIALES",
    "TECNICO DE NIVEL SUPERIOR EN TRANSPORTE MARITIMO",
    "TECNICO EN ADMINISTRACION DE EMPRESAS MENCION LOGISTICA",
    "TECNICO EN COMERCIO EXTERIOR",
    "TECNICO EN GESTION LOGISTICA",
    "TECNICO EN LOGISTICA",
    "TECNICO EN LOGISTICA MARITIMA PORTUARIA",
    "TECNICO EN OPERACIONES LOGISTICAS",
    "TECNICO UNIVERSITARIO EN LOGISTICA"
]

d.loc[
    d['NOMBRE CARRERA'].isin(logistica),
    'ADN_BIOBIO'
] = 'Logistica y puertos'


#4. Forestal y madera
forestal = [
    "INGENIERIA CIVIL EN INDUSTRIAS DE LA MADERA",
    "INGENIERIA FORESTAL"
]

d.loc[
    d['NOMBRE CARRERA'].isin(forestal),
    'ADN_BIOBIO'
] = 'Forestal y madera'


#5. Pesca y acuicultura
pesca = [
    "ADMINISTRACION PORTUARIA PESQUERO ARTESANAL",
    "GESTION PORTUARIA Y PESQUERA",
    "INGENIERIA EN BIOTECNOLOGIA MARINA Y ACUICULTURA"
]

d.loc[
    d['NOMBRE CARRERA'].isin(pesca),
    'ADN_BIOBIO'
] = 'Pesca y acuicultura'


#6. Agroalimentario
agro = [
    "AGRONOMIA",
    "AGRONOMIA, PLAN CONTINUIDAD DE ESTUDIOS",
    "INGENIERIA AGRICOLA",
    "INGENIERIA DE EJECUCION AGROPECUARIA",
    "TECNICO AGRICOLA Y GANADERO",
    "TECNICO DE NIVEL SUPERIOR EN GESTION AGROINDUSTRIAL",
    "TECNICO EN VETERINARIA Y PRODUCCION PECUARIA",
    "TECNICO UNIVERSITARIO AGROPECUARIO",
    "TECNICO UNIVERSITARIO EN ALIMENTOS"
]

d.loc[
    d['NOMBRE CARRERA'].isin(agro),
    'ADN_BIOBIO'
] = 'Agroalimentario'


**Análisis e interpretación:** Las carreras vinculadas con la base productiva regional se agruparon en seis familias: **Industria y manufactura, Construcción e infraestructura, Logística y puertos, Forestal y madera, Pesca y acuicultura y Agroalimentario**. Para realizar la asignación se definieron listas de carreras y se verificó su pertenencia mediante `isin()`. Esta clasificación fue elaborada específicamente para el proyecto con el propósito de representar actividades productivas relevantes para el análisis regional; por lo tanto, debe entenderse como una decisión metodológica del grupo y no como una clasificación oficial de las carreras.


In [21]:
#Clasificamos las familias asociadas a capacidades transformadoras

#7. Digital y TIC
digital = [
    "ANALISTA PROGRAMADOR",
    "ANALISTA PROGRAMADOR COMPUTACIONAL",
    "COMPUTACION E INFORMATICA",
    "INFORMATICA BIOMEDICA",
    "INGENIERIA CIVIL EN INFORMATICA",
    "INGENIERIA CIVIL EN TELECOMUNICACIONES",
    "INGENIERIA CIVIL INFORMATICA",
    "INGENIERIA DE EJECUCION EN COMPUTACION E INFORMATICA",
    "INGENIERIA DE EJECUCION EN COMPUTACION E INFORMATICA-PE",
    "INGENIERIA DE EJECUCION EN INFORMATICA MENCION DESARROLLO DE SISTEMAS",
    "INGENIERIA EN CIBERSEGURIDAD",
    "INGENIERIA EN CONECTIVIDAD Y REDES",
    "INGENIERIA EN INFORMATICA",
    "INGENIERIA EN TELECOMUNICACIONES, CONECTIVIDAD Y REDES",
    "INGENIERIA GEOMATICA",
    "INGENIERIA INFORMATICA",
    "PROGRAMACION COMPUTACIONAL",
    "PROGRAMACION Y ANALISIS DE SISTEMAS",
    "TECNICO ANALISTA PROGRAMADOR",
    "TECNICO EN CONECTIVIDAD Y REDES",
    "TECNICO EN PROGRAMACION Y ANALISIS DE SISTEMAS",
    "TECNICO UNIVERSITARIO EN INFORMATICA",
    "TECNICO UNIVERSITARIO EN TELECOMUNICACIONES Y REDES",
    "TELECOMUNICACIONES, CONECTIVIDAD Y REDES"
]

d.loc[
    d['NOMBRE CARRERA'].isin(digital),
    'ADN_BIOBIO'
] = 'Digital y TIC'


#8. Automatización y robótica
automatizacion = [
    "AUTOMATIZACION Y CONTROL INDUSTRIAL",
    "ELECTRONICA",
    "ELECTRONICA INDUSTRIAL",
    "INGENIERIA CIVIL ELECTRONICA",
    "INGENIERIA CIVIL EN AUTOMATIZACION",
    "INGENIERIA DE EJECUCION ELECTRONICA",
    "INGENIERIA DE EJECUCION EN AUTOMATIZACION Y CONTROL-PE",
    "INGENIERIA DE EJECUCION EN CONTROL E INSTRUMENTACION INDUSTRIAL",
    "INGENIERIA DE EJECUCION EN ELECTRONICA",
    "INGENIERIA ELECTRONICA",
    "INGENIERIA EN AUTOMATIZACION Y CONTROL INDUSTRIAL",
    "INGENIERIA EN AUTOMATIZACION Y ROBOTICA",
    "INGENIERIA EN ELECTRICIDAD Y AUTOMATIZACION INDUSTRIAL",
    "INGENIERIA EN ELECTRICIDAD Y ELECTRONICA",
    "INGENIERIA EN ELECTRONICA Y SISTEMAS INTELIGENTES",
    "INSTRUMENTACION Y AUTOMATIZACION INDUSTRIAL",
    "INSTRUMENTACION Y CONTROL INDUSTRIAL",
    "TECNICO EN AUTOMATIZACION Y CONTROL INDUSTRIAL",
    "TECNICO EN AUTOMATIZACION Y ROBOTICA",
    "TECNICO EN ELECTRICIDAD Y AUTOMATIZACION INDUSTRIAL",
    "TECNICO EN ELECTRICIDAD Y ELECTRONICA",
    "TECNICO EN INSTRUMENTACION Y CONTROL",
    "TECNICO UNIVERSITARIO EN AUTOMATIZACION INDUSTRIAL",
    "TECNICO UNIVERSITARIO EN AUTOMATIZACION Y CONTROL",
    "TECNICO UNIVERSITARIO EN ELECTRONICA",
    "TECNICO UNIVERSITARIO EN ROBOTICA Y MECATRONICA"
]

d.loc[
    d['NOMBRE CARRERA'].isin(automatizacion),
    'ADN_BIOBIO'
] = 'Automatizacion y robotica'


#9. Energía, sustentabilidad y biotecnología
energia = [
    "ELECTRICIDAD Y EFICIENCIA ENERGETICA",
    "INGENIERIA AMBIENTAL",
    "INGENIERIA DE EJECUCION EN ENERGIAS RENOVABLES Y SUSTENTABILIDAD AMBIENTAL",
    "INGENIERIA DE EJECUCION EN GESTION Y CONTROL AMBIENTAL",
    "INGENIERIA DE EJECUCION EN QUIMICA Y MEDIO AMBIENTE-PE",
    "INGENIERIA EN BIOTECNOLOGIA VEGETAL",
    "INGENIERIA EN ELECTRICIDAD Y ENERGIAS RENOVABLES",
    "INGENIERIA EN ENERGIA Y SUSTENTABILIDAD AMBIENTAL",
    "INGENIERIA EN PREVENCION DE RIESGO Y MEDIOAMBIENTE",
    "QUIMICA AMBIENTAL",
    "TECNICO EN ENERGIAS RENOVABLES Y EFICIENCIA ENERGETICA",
    "TECNICO EN MEDIOAMBIENTE",
    "TECNICO UNIVERSITARIO EN CONTROL MEDIO AMBIENTE"
]

d.loc[
    d['NOMBRE CARRERA'].isin(energia),
    'ADN_BIOBIO'
] = 'Energia, sustentabilidad y biotecnologia'


**Análisis e interpretación:** Las capacidades asociadas a transformación tecnológica se organizaron en tres familias: **Digital y TIC, Automatización y robótica, y Energía, sustentabilidad y biotecnología**. Estas categorías reúnen carreras relacionadas con digitalización, automatización, tecnologías de información, energía y nuevas capacidades aplicables a procesos productivos. Al igual que en el bloque anterior, la agrupación fue construida para responder la pregunta central del proyecto y se utiliza como marco analítico, no como una clasificación oficial externa.


In [22]:
#Creamos los dos grandes bloques del análisis
motores = [
    'Industria y manufactura',
    'Construccion e infraestructura',
    'Logistica y puertos',
    'Forestal y madera',
    'Pesca y acuicultura',
    'Agroalimentario'
]

transformadoras = [
    'Digital y TIC',
    'Automatizacion y robotica',
    'Energia, sustentabilidad y biotecnologia'
]

d['BLOQUE_ADN'] = 'Otros campos'

d.loc[
    d['ADN_BIOBIO'].isin(motores),
    'BLOQUE_ADN'
] = 'Motores productivos'

d.loc[
    d['ADN_BIOBIO'].isin(transformadoras),
    'BLOQUE_ADN'
] = 'Capacidades transformadoras'


**Análisis e interpretación:** Una vez clasificadas las carreras en nueve familias, estas se resumieron en tres grandes categorías: **Motores productivos, Capacidades transformadoras y Otros campos**. La creación de la variable **BLOQUE_ADN** permite comparar directamente el peso de la formación vinculada con la base productiva frente a la formación asociada con capacidades tecnológicas. La categoría “Otros campos” se mantiene porque gran parte de la educación superior regional pertenece a áreas que no forman parte de los dos bloques definidos para la pregunta central.


In [23]:
#Calculamos la frecuencia de los tres bloques
cuenta_bloque = d.groupby('BLOQUE_ADN').size()

tabla_bloque = pd.DataFrame({
    'Frecuencia absoluta': cuenta_bloque,
    'Frecuencia relativa (%)': cuenta_bloque/d.shape[0] * 100
})

round(tabla_bloque, 2)


,Frecuencia absoluta,Frecuencia relativa (%)
BLOQUE_ADN,,
Capacidades transformadoras,7000,6.92
Motores productivos,20368,20.15
Otros campos,73725,72.93


**Análisis e interpretación:** La clasificación final ubica **20.368 matrículas (20,15%)** en Motores productivos, **7.000 (6,92%)** en Capacidades transformadoras y **73.725 (72,93%)** en Otros campos. El resultado muestra que, dentro del marco creado para este trabajo, el bloque productivo tiene un peso considerablemente mayor que el transformador. Al mismo tiempo, el 72,93% correspondiente a “Otros campos” recuerda que esta clasificación no pretende describir toda la educación superior, sino enfocarse únicamente en las áreas relacionadas con la pregunta “Del acero al algoritmo”.


In [24]:
#Comparamos los dos bloques principales
tabla_acero_algoritmo = tabla_bloque.loc[
    ['Motores productivos', 'Capacidades transformadoras']
]

fig, ax = plt.subplots(figsize=(7,5))

ax.bar(
    tabla_acero_algoritmo.index,
    tabla_acero_algoritmo['Frecuencia relativa (%)'],
    edgecolor='black'
)

ax.set_title('Del acero al algoritmo')
ax.set_ylabel('Porcentaje de matrículas')

plt.show()


**Análisis e interpretación:** La comparación gráfica entre Motores productivos y Capacidades transformadoras muestra una diferencia clara: **20,15%** de las matrículas se vincula al bloque productivo y **6,92%** al transformador. La visualización facilita apreciar la magnitud de la distancia entre ambos grupos. Este resultado describe la composición de la matrícula de pregrado en 2021 dentro de nuestra clasificación, pero no permite concluir por sí solo que exista un déficit tecnológico, una falta de profesionales o una condición de atraso regional.


In [25]:
#Calculamos cuántas veces es mayor el bloque productivo
razon = (
    tabla_acero_algoritmo.loc[
        'Motores productivos',
        'Frecuencia absoluta'
    ]
    /
    tabla_acero_algoritmo.loc[
        'Capacidades transformadoras',
        'Frecuencia absoluta'
    ]
)

round(razon, 1)


np.float64(2.9)

**Análisis e interpretación:** La relación entre ambos bloques se calculó dividiendo las **20.368 matrículas de Motores productivos** por las **7.000 de Capacidades transformadoras**, obteniéndose aproximadamente **2,9**. Esto significa que, dentro de la clasificación propuesta, existen cerca de **2,9 matrículas productivas por cada matrícula transformadora**. La razón resume en un solo indicador la diferencia entre los dos bloques y facilita comunicar el hallazgo central sin reemplazar los porcentajes originales.


In [26]:
#Revisamos las familias estratégicas por separado
cuenta_adn = d[
    d['ADN_BIOBIO'] != 'Otros campos'
].groupby('ADN_BIOBIO').size()

tabla_adn = pd.DataFrame({
    'Frecuencia absoluta': cuenta_adn,
    'Frecuencia relativa (%)': cuenta_adn/d.shape[0] * 100
})

tabla_adn = tabla_adn.sort_values(
    'Frecuencia absoluta',
    ascending=False
)

round(tabla_adn, 2)


,Frecuencia absoluta,Frecuencia relativa (%)
ADN_BIOBIO,,
Industria y manufactura,11221,11.10
Construccion e infraestructura,6916,6.84
Digital y TIC,3365,3.33
Automatizacion y robotica,2608,2.58
Logistica y puertos,1046,1.03
"Energia, sustentabilidad y biotecnologia",1027,1.02
Agroalimentario,863,0.85
Forestal y madera,171,0.17
Pesca y acuicultura,151,0.15


**Análisis e interpretación:** Al desagregar los dos bloques en sus nueve familias se observa que **Industria y manufactura** concentra **11.221 matrículas (11,10%)**, seguida por **Construcción e infraestructura (6,84%)**. Dentro de las capacidades transformadoras, **Digital y TIC representa 3,33%**, **Automatización y robótica 2,58%** y **Energía, sustentabilidad y biotecnología 1,02%**. Las demás familias productivas presentan participaciones menores. Esta desagregación muestra que la diferencia entre los dos grandes bloques no se distribuye de manera uniforme, sino que está explicada principalmente por algunas familias con mayor cantidad de matrículas.


### Resultado principal

En 2021, **20,15%** de las matrículas de pregrado queda asociado a motores productivos y **6,92%** a capacidades transformadoras.

La relación es cercana a **2,9 a 1**. Esto describe la composición de la matrícula de 2021, pero no demuestra que exista déficit de profesionales.


# 5. Preguntas del trabajo


## Pregunta 1
### ¿Hay áreas del conocimiento donde las carreras sean más caras?

El análisis se realiza sobre las **1.273 ofertas académicas únicas**, de manera que cada oferta tenga un peso comparable.

**Criterio diseñado:** se compara la **mediana del arancel de cada área** con la **mediana global** de las ofertas. Se utiliza la mediana porque es menos sensible a valores extremos y representa mejor el arancel típico de un área.


In [27]:
#Agrupamos las ofertas por área del conocimiento
datos_p1 = ofertas.groupby(
    'AREA CONOCIMIENTO'
)['VALOR ARANCEL (PESOS)']

#Calculamos cantidad, media, mediana y cuartiles
tabla_p1 = pd.DataFrame({
    'Cantidad de ofertas': datos_p1.size(),
    'Media': datos_p1.mean(),
    'Mediana': datos_p1.median(),
    'Percentil 25': datos_p1.quantile(0.25),
    'Percentil 75': datos_p1.quantile(0.75)
})

tabla_p1 = tabla_p1.sort_values(
    'Mediana',
    ascending=False
)

mediana_global = ofertas[
    'VALOR ARANCEL (PESOS)'
].median()

print(f'Mediana global: ${mediana_global:.0f}')
round(tabla_p1, 0)


Mediana global: $2030000


,Cantidad de ofertas,Media,Mediana,Percentil 25,Percentil 75
AREA CONOCIMIENTO,,,,,
Derecho,18,3319065.0,3586000.0,2350000.0,4215000.0
Ciencias Basicas,31,3256166.0,3290000.0,2194500.0,4278045.0
Agropecuaria,18,3150941.0,2981500.0,1824250.0,4335000.0
Tecnologia,491,2301377.0,2122000.0,1730000.0,2276000.0
Salud,131,3044197.0,2080000.0,1781000.0,4203000.0
Humanidades,8,2115167.0,2006500.0,1717500.0,2579904.0
Arte y Arquitectura,35,2452345.0,1940000.0,1786000.0,2826500.0
Administracion y Comercio,251,2031019.0,1855000.0,1537000.0,2122000.0
Educacion,174,1952592.0,1722500.0,1399000.0,2654850.0


**Análisis e interpretación:** Para responder si existen áreas donde las carreras sean más caras se trabajó con las **1.273 ofertas académicas únicas**, de modo que cada oferta tuviera un peso comparable. Por área se calcularon cantidad de ofertas, media, mediana, percentil 25 y percentil 75, utilizando como referencia una **mediana global de \$2.030.000**. Según el criterio definido, las áreas cuya mediana supera ese valor son **Derecho (\$3.586.000)**, **Ciencias Básicas (\$3.290.000)**, **Agropecuaria (\$2.981.500)**, **Tecnología (\$2.122.000)** y **Salud (\$2.080.000)**. Derecho, Ciencias Básicas y Agropecuaria presentan las diferencias más amplias respecto de la mediana global, mientras que Tecnología y Salud la superan por un margen menor. La mediana se utiliza como medida principal porque representa el centro de cada distribución y reduce la influencia de ofertas excepcionalmente altas o bajas.


In [28]:
#Graficamos la mediana del arancel por área
fig, ax = plt.subplots(figsize=(10,5))

ax.bar(
    tabla_p1.index,
    tabla_p1['Mediana']/1000000,
    edgecolor='black'
)

ax.set_title('Arancel mediano por área del conocimiento')
ax.set_ylabel('Millones de pesos')
plt.xticks(rotation=45, ha='right')

plt.show()


**Análisis e interpretación:** El gráfico ordena las áreas según su arancel mediano y permite comparar visualmente su posición respecto de la mediana global de **\$2.030.000**. Las medianas de **Derecho, Ciencias Básicas, Agropecuaria, Tecnología y Salud** se encuentran por encima de esa referencia. Entre ellas, Derecho, Ciencias Básicas y Agropecuaria destacan con las mayores diferencias. Por lo tanto, la respuesta a la primera pregunta es afirmativa: el arancel típico no es igual entre las áreas del conocimiento. Esta comparación es descriptiva y no permite atribuir las diferencias exclusivamente al área, por lo que se incorpora posteriormente el tipo de institución como variable complementaria.


In [29]:
#Cruzamos área del conocimiento y tipo de institución
tabla_p1_bivariada = pd.crosstab(
    ofertas['AREA CONOCIMIENTO'],
    ofertas['TIPO DE INSTITUCION'],
    ofertas['VALOR ARANCEL (PESOS)'],
    aggfunc='mean',
    margins=True,
    margins_name='Total'
)

round(tabla_p1_bivariada, 0)


TIPO DE INSTITUCION,Centros de Formacion Tecnica,Institutos Profesionales,Universidades (* Carrera en Convenio),Universidades CRUCH,Universidades Privadas,Total
AREA CONOCIMIENTO,,,,,,
Administracion y Comercio,1781338.0,1741044.0,NaN,2473294.0,2962050.0,2031019.0
Agropecuaria,1679333.0,1880000.0,NaN,4059110.0,4340417.0,3150941.0
Arte y Arquitectura,1769000.0,1850750.0,NaN,3848769.0,4637620.0,2452345.0
Ciencias Basicas,NaN,1878875.0,NaN,3758102.0,3495000.0,3256166.0
Ciencias Sociales,1353272.0,1419590.0,2327000.0,3691376.0,3069591.0,2129750.0
Derecho,NaN,1750000.0,2229000.0,4231569.0,3503926.0,3319065.0
Educacion,1364772.0,1439560.0,NaN,2673509.0,2490693.0,1952592.0
Humanidades,NaN,1005000.0,NaN,2763334.0,1929000.0,2115167.0
Salud,1735035.0,1740270.0,NaN,4419709.0,4246702.0,3044197.0


**Análisis e interpretación:** Como análisis complementario, la tabla bivariada cruza **área del conocimiento** con **tipo de institución** y calcula el arancel promedio para cada combinación disponible. Se observan diferencias entre tipos de institución incluso dentro de una misma área, lo que indica que la variación de precios no queda completamente resumida por el área del conocimiento. Este cruce se utiliza como apoyo exploratorio; el criterio principal de la Pregunta 1 continúa siendo la comparación de **medianas por área** con la mediana global, ya que la media puede verse más afectada por valores extremos.


### Respuesta

Sí. La mediana global de las **1.273 ofertas académicas únicas** es **\$2.030.000**. De acuerdo con el criterio definido, presentan una mediana superior a ese valor **Derecho (\$3.586.000)**, **Ciencias Básicas (\$3.290.000)**, **Agropecuaria (\$2.981.500)**, **Tecnología (\$2.122.000)** y **Salud (\$2.080.000)**. Las tres primeras muestran las diferencias más marcadas respecto de la referencia global.

La tabla bivariada también muestra diferencias de arancel entre tipos de institución dentro de una misma área. Por esta razón, el área del conocimiento se interpreta como una variable asociada a diferencias de precio, pero no como la única característica relacionada con el arancel.


## Pregunta 2
### ¿Qué relación se observa entre la edad y el tipo de institución?

El análisis comienza con un resumen de la edad por tipo de institución y luego compara la distribución institucional de dos grupos etarios contrastantes: **15 a 19 años** y **40 años o más**. La comparación se complementa con una tabla de contingencia que incluye todos los rangos de edad.


In [30]:
#Calculamos cantidad, media y mediana de edad por tipo de institución
tabla_edad_institucion = d.groupby(
    'TIPO DE INSTITUCION'
)['EDAD'].agg(
    ['count', 'mean', 'median']
)

round(tabla_edad_institucion, 1)


,count,mean,median
TIPO DE INSTITUCION,,,
Centros de Formacion Tecnica,9455,25.7,23.0
Institutos Profesionales,26724,26.5,24.0
Universidades (* Carrera en Convenio),9,46.0,48.0
Universidades CRUCH,37997,22.8,22.0
Universidades Privadas,26908,24.0,23.0


**Análisis e interpretación:** La comparación inicial por tipo de institución muestra diferencias en la composición etaria. Las **Universidades CRUCH** presentan una edad media de **22,8 años** y mediana de 22, mientras que los **Institutos Profesionales** alcanzan una media de **26,5 años** y mediana de 24. Los CFT presentan una media de 25,7 años y las universidades privadas de 24 años. El grupo “Universidades (* Carrera en Convenio)” contiene solo 9 registros, por lo que su resultado no se utiliza para formular conclusiones generales. Estas diferencias justifican profundizar el análisis mediante rangos de edad y porcentajes por tipo de institución.


In [31]:
#Creamos una tabla de contingencia entre rango de edad e institución
tabla_edad_inst = pd.crosstab(
    d['RANGO EDAD'],
    d['TIPO DE INSTITUCION'],
    margins=True,
    margins_name='Total'
)

tabla_edad_inst


TIPO DE INSTITUCION,Centros de Formacion Tecnica,Institutos Profesionales,Universidades (* Carrera en Convenio),Universidades CRUCH,Universidades Privadas,Total
RANGO EDAD,,,,,,
15 a 19,1847,3570,0,8554,4527,18498
20 a 24,4075,11633,0,21963,14724,52395
25 a 29,1467,4842,0,5409,4826,16544
30 a 34,894,2826,2,1173,1477,6372
35 a 39,496,1829,1,482,652,3460
40 y mas,676,2024,6,416,702,3824
Total,9455,26724,9,37997,26908,101093


**Análisis e interpretación:** La tabla de contingencia permite observar simultáneamente los **rangos de edad** y el **tipo de institución**. Las frecuencias muestran que la composición institucional no se mantiene constante entre grupos etarios; por ejemplo, en el rango de 15 a 19 años existen **8.554 matrículas en universidades CRUCH**, mientras que entre las personas de 40 años o más el número se reduce a **416**. Sin embargo, para comparar grupos de distinto tamaño resulta más adecuado complementar estas cantidades con porcentajes, lo que se realiza en los análisis siguientes.


In [32]:
#Analizamos el grupo de 15 a 19 años
grupo_15_19 = d[
    d['RANGO EDAD'] == '15 a 19 '
]

frecuencia_15_19 = grupo_15_19.groupby(
    'TIPO DE INSTITUCION'
).size()

tabla_15_19 = pd.DataFrame({
    'Frecuencia absoluta': frecuencia_15_19,
    'Frecuencia relativa (%)':
        frecuencia_15_19/grupo_15_19.shape[0] * 100
})

round(tabla_15_19, 1)


,Frecuencia absoluta,Frecuencia relativa (%)
TIPO DE INSTITUCION,,
Centros de Formacion Tecnica,1847,10.0
Institutos Profesionales,3570,19.3
Universidades CRUCH,8554,46.2
Universidades Privadas,4527,24.5


**Análisis e interpretación:** Dentro del grupo de **15 a 19 años**, las universidades CRUCH concentran **46,2%** de las matrículas, seguidas por universidades privadas con **24,5%**, Institutos Profesionales con **19,3%** y CFT con **10,0%**. En conjunto, las dos categorías universitarias reúnen aproximadamente **70,7%** del grupo más joven. Este resultado muestra que, entre las matrículas de 15 a 19 años, la participación universitaria es claramente mayor y sirve como punto de comparación con los grupos de edad superiores.


In [33]:
#Analizamos el grupo de 40 años o más
grupo_40 = d[
    d['RANGO EDAD'] == '40 y mas'
]

frecuencia_40 = grupo_40.groupby(
    'TIPO DE INSTITUCION'
).size()

tabla_40 = pd.DataFrame({
    'Frecuencia absoluta': frecuencia_40,
    'Frecuencia relativa (%)':
        frecuencia_40/grupo_40.shape[0] * 100
})

round(tabla_40, 1)


,Frecuencia absoluta,Frecuencia relativa (%)
TIPO DE INSTITUCION,,
Centros de Formacion Tecnica,676,17.7
Institutos Profesionales,2024,52.9
Universidades (* Carrera en Convenio),6,0.2
Universidades CRUCH,416,10.9
Universidades Privadas,702,18.4


**Análisis e interpretación:** En las personas de **40 años o más** la distribución institucional cambia de manera importante. Los **Institutos Profesionales representan 52,9%** del grupo, mientras que las universidades privadas alcanzan 18,4%, los CFT 17,7%, las universidades CRUCH 10,9% y las carreras en convenio 0,2%. Al compararlo con el grupo de 15 a 19 años se observa un cambio claro: los IP pasan de **19,3% a 52,9%**, mientras que las universidades CRUCH disminuyen de **46,2% a 10,9%**. Esto evidencia una asociación entre el grupo etario y la composición institucional.


In [34]:
#Comparamos los dos grupos con gráficos de barras
fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(13,5))

ax1.bar(
    tabla_15_19.index,
    tabla_15_19['Frecuencia relativa (%)'],
    edgecolor='black'
)

ax2.bar(
    tabla_40.index,
    tabla_40['Frecuencia relativa (%)'],
    edgecolor='black'
)

ax1.set_title('15 a 19 años')
ax2.set_title('40 años o más')
ax1.set_ylabel('Frecuencia relativa (%)')
ax2.set_ylabel('Frecuencia relativa (%)')

ax1.tick_params(axis='x', rotation=45)
ax2.tick_params(axis='x', rotation=45)

plt.tight_layout()
plt.show()


**Análisis e interpretación:** La comparación gráfica de los grupos de **15 a 19 años** y **40 años o más** resume el patrón observado en las tablas. En los estudiantes más jóvenes predominan las universidades CRUCH, mientras que en el grupo de mayor edad predominan los Institutos Profesionales. Esto permite afirmar que existe una **asociación descriptiva entre edad y tipo de institución** en la base analizada. No obstante, los datos disponibles no permiten establecer que la edad sea la causa de la elección institucional, ya que no se observan factores como preferencias personales, situación laboral, ingresos o motivos de ingreso.


### Respuesta

Entre **15 y 19 años**, las universidades CRUCH representan aproximadamente **46,2%** del grupo y los Institutos Profesionales **19,3%**.

Entre quienes tienen **40 años o más**, los Institutos Profesionales representan aproximadamente **52,9%** y las universidades CRUCH **10,9%**.

Se observa una asociación entre edad y tipo de institución, pero la base no permite afirmar que la edad sea la causa de la elección.


## Pregunta 3
### ¿Hay carreras cuyo arancel sea sustantivamente más caro que la mayoría?

Para responder esta pregunta se definió como arancel sustantivamente alto aquel que se encuentre **por sobre el percentil 90 (P90)** de las ofertas académicas únicas.

Este criterio permite separar de manera reproducible aproximadamente el **10% superior de la distribución de aranceles** y analizar posteriormente qué características presentan esas ofertas.


In [35]:
#Calculamos el percentil 90 de las ofertas académicas
p90_ofertas = ofertas[
    'VALOR ARANCEL (PESOS)'
].quantile(0.90)

#También mostramos el valor máximo
maximo_ofertas = ofertas[
    'VALOR ARANCEL (PESOS)'
].max()

print(f'Percentil 90: ${p90_ofertas:.0f}')
print(f'Máximo: ${maximo_ofertas:.0f}')


Percentil 90: $4106400
Máximo: $8783670


**Análisis e interpretación:** Para identificar ofertas con arancel sustantivamente alto se definió como umbral el **percentil 90 (P90)**. El valor obtenido es aproximadamente **\$4.106.400**, lo que significa que cerca del 90% de las ofertas académicas tiene un arancel igual o inferior a ese monto y que el grupo situado por encima corresponde aproximadamente al 10% superior. Este criterio permite delimitar de manera reproducible el conjunto de aranceles altos sin fijar un monto arbitrario.


In [36]:
#Filtramos las ofertas que superan el percentil 90
altas = ofertas[
    ofertas['VALOR ARANCEL (PESOS)'] > p90_ofertas
].copy()

cantidad_altas = altas.shape[0]
porcentaje_altas = cantidad_altas / ofertas.shape[0] * 100

print('Ofertas sobre P90:', cantidad_altas)
print(f'Porcentaje: {porcentaje_altas:.1f}%')


Ofertas sobre P90: 128
Porcentaje: 10.1%


**Análisis e interpretación:** Al aplicar el umbral del P90 se identifican **128 ofertas de un total de 1.273**, equivalentes a aproximadamente **10,1%**. La proporción es coherente con el objetivo del percentil 90, que busca aislar la parte superior de la distribución. A partir de este grupo se analizan características como tipo de institución, área del conocimiento, provincia y duración, con el propósito de describir qué rasgos presentan las ofertas de mayor arancel.


In [37]:
#Revisamos dónde se concentran las ofertas de arancel alto
print('Por tipo de institución')
print(
    altas.groupby('TIPO DE INSTITUCION')
    .size()
    .sort_values(ascending=False)
)

print('\nPor área del conocimiento')
print(
    altas.groupby('AREA CONOCIMIENTO')
    .size()
    .sort_values(ascending=False)
)

print('\nPor provincia')
print(
    altas.groupby('PROVINCIA SEDE')
    .size()
    .sort_values(ascending=False)
)


Por tipo de institución
TIPO DE INSTITUCION
Universidades CRUCH       73
Universidades Privadas    55
dtype: int64

Por área del conocimiento
AREA CONOCIMIENTO
Salud                        38
Tecnologia                   38
Ciencias Basicas             13
Administracion y Comercio    10
Ciencias Sociales            10
Agropecuaria                  7
Derecho                       7
Arte y Arquitectura           5
dtype: int64

Por provincia
PROVINCIA SEDE
Concepcion    123
Biobio          5
dtype: int64


**Análisis e interpretación:** Entre las **128 ofertas sobre el P90**, **73 corresponden a universidades CRUCH y 55 a universidades privadas**; en esta base, por lo tanto, todas las ofertas que superan el umbral pertenecen a universidades. Por área del conocimiento, **Salud y Tecnología concentran 38 ofertas cada una**, es decir, en conjunto reúnen aproximadamente **59,4%** del grupo de arancel alto. Territorialmente, **123 ofertas se localizan en la provincia de Concepción y 5 en Biobío**, por lo que Concepción concentra aproximadamente **96,1%** del grupo. Estos resultados permiten caracterizar con mayor precisión las ofertas sobre P90, pero siguen siendo asociaciones descriptivas y no demuestran que el tipo de institución, el área o la ubicación sean la causa directa del precio.


In [38]:
#Comparamos la duración mediana de las ofertas altas con el resto
duracion_altas = altas[
    'DURACION TOTAL CARRERA (SEMESTRES)'
].median()

resto = ofertas[
    ofertas['VALOR ARANCEL (PESOS)'] <= p90_ofertas
]

duracion_resto = resto[
    'DURACION TOTAL CARRERA (SEMESTRES)'
].median()

print(f'Duración mediana ofertas altas: {duracion_altas} semestres')
print(f'Duración mediana resto: {duracion_resto} semestres')


Duración mediana ofertas altas: 10.0 semestres
Duración mediana resto: 5.0 semestres


**Análisis e interpretación:** La duración también presenta una diferencia marcada entre grupos. Las ofertas con arancel sobre el P90 tienen una **duración mediana de 10 semestres**, mientras que en el resto de las ofertas la mediana es de **5 semestres**. En términos descriptivos, la mediana del grupo de arancel alto duplica la del resto, por lo que una mayor duración aparece asociada con las ofertas de precios superiores. Sin embargo, esta comparación no permite concluir que la duración provoque el mayor arancel, ya que pueden intervenir otras características de las carreras y de las instituciones.


In [39]:
#Calculamos el arancel mediano por carrera
ranking_carreras = ofertas.groupby(
    'NOMBRE CARRERA'
)['VALOR ARANCEL (PESOS)'].agg(
    ['count', 'median']
)

ranking_carreras.columns = [
    'Cantidad de ofertas',
    'Arancel mediano'
]

#Consideramos carreras con al menos 3 ofertas
ranking_carreras = ranking_carreras[
    ranking_carreras['Cantidad de ofertas'] >= 3
]

ranking_carreras = ranking_carreras.sort_values(
    'Arancel mediano',
    ascending=False
).head(10)

ranking_carreras


,Cantidad de ofertas,Arancel mediano
NOMBRE CARRERA,,
MEDICINA,3,7490000.0
ODONTOLOGIA,5,7290000.0
INGENIERIA CIVIL QUIMICA,3,4804479.0
TECNOLOGIA MEDICA,4,4771050.0
INGENIERIA CIVIL INFORMATICA,3,4745935.0
MEDICINA VETERINARIA,6,4320000.0
KINESIOLOGIA,9,4224000.0
ARQUITECTURA,6,4214450.0
INGENIERIA CIVIL ELECTRICA,4,4202239.5


**Análisis e interpretación:** Para identificar carreras con aranceles típicos altos se calculó, sobre las **1.273 ofertas únicas**, la cantidad de ofertas y el arancel mediano de cada carrera. Se conservaron únicamente las carreras con **al menos tres ofertas**, con el fin de evitar que una sola observación determine el resultado. **Medicina** aparece con 3 ofertas y una mediana aproximada de **\$7.490.000**, mientras que **Odontología** tiene 5 ofertas y una mediana de **\$7.290.000**. También aparecen varias ingenierías y Tecnología Médica entre las mayores medianas. Este ranking se refiere a la **mediana por carrera** y complementa el análisis del P90; no significa que todas las ofertas individuales de una carrera tengan exactamente ese precio.


### Respuesta

Sí. Para identificar aranceles sustantivamente altos se utilizó el **percentil 90**, cuyo valor en las 1.273 ofertas académicas únicas es aproximadamente **\$4.106.400**. Sobre ese umbral se encuentran **128 ofertas**, equivalentes a **10,1%** del total.

Las 128 ofertas pertenecen a universidades: **73 a instituciones CRUCH y 55 a universidades privadas**. **Salud** y **Tecnología** reúnen 38 ofertas cada una, por lo que juntas concentran aproximadamente **59,4%** del grupo. Además, **123 de las 128 ofertas (96,1%)** se ubican en la provincia de **Concepción** y 5 en Biobío.

La duración mediana también diferencia a ambos grupos: las ofertas sobre P90 presentan **10 semestres**, frente a **5 semestres** en el resto. Al observar las carreras con al menos tres ofertas, las mayores medianas corresponden a **Medicina (aprox. \$7.490.000)** y **Odontología (\$7.290.000)**.

**Interpretación:** las ofertas de mayor arancel se caracterizan, dentro de esta base, por concentrarse en universidades, especialmente en las áreas de Salud y Tecnología, por tener una duración mediana mayor y por localizarse principalmente en sedes de Concepción. Estos patrones ayudan a describir el grupo de arancel alto, pero no permiten afirmar que esas variables sean la causa directa del precio.


# 6. Conclusiones

- La base original contiene **106.555 registros** y el análisis principal utiliza **101.093 matrículas de pregrado**. Para los cálculos generales de arancel se consideran **101.067 matrículas con arancel mayor que cero**, mientras que las comparaciones de precios entre áreas y carreras se realizan con **1.273 ofertas académicas únicas**.
- Dentro de la clasificación creada para “Del acero al algoritmo”, los **Motores productivos representan 20,15%** de las matrículas de pregrado y las **Capacidades transformadoras 6,92%**, con una relación aproximada de **2,9 a 1**. Esta comparación describe la composición de 2021 y no constituye una medida de déficit laboral.
- En la **Pregunta 1**, la mediana global de las ofertas es **\$2.030.000**. Las áreas con mediana superior son **Derecho, Ciencias Básicas, Agropecuaria, Tecnología y Salud**, destacando las tres primeras con las mayores diferencias.
- En la **Pregunta 2**, la composición institucional cambia según la edad. Entre 15 y 19 años las universidades CRUCH representan **46,2%**, mientras que en 40 años o más los Institutos Profesionales alcanzan **52,9%**. El resultado evidencia asociación, no causalidad.
- En la **Pregunta 3**, el **P90 es aproximadamente \$4.106.400** y **128 ofertas (10,1%)** quedan sobre ese umbral. Todas pertenecen a universidades, **59,4%** se concentra entre Salud y Tecnología y **96,1%** se localiza en sedes de la provincia de Concepción. La duración mediana es de 10 semestres, frente a 5 en el resto.

# 7. Limitaciones

- La base corresponde únicamente a **2021**, por lo que no permite estudiar evolución temporal.
- La clasificación “Del acero al algoritmo” fue construida para este trabajo y no corresponde a una clasificación oficial.
- Los resultados son **descriptivos** y permiten identificar diferencias y asociaciones, pero no demostrar relaciones causales.
- La variable **PROVINCIA SEDE** corresponde a la ubicación de la sede de estudio y no necesariamente al lugar de residencia del estudiante.
- La base no contiene salarios, vacantes, productividad, demanda laboral ni motivaciones de elección, por lo que esas dimensiones no pueden inferirse a partir de estos datos.
